File: packages/ruby/model/arena.rb
class Arena
    attr_reader :width
    attr_reader :height
    attr_reader :depth
    attr_reader :bottom_radius
    attr_reader :top_radius
    attr_reader :corner_radius
    attr_reader :goal_top_radius
    attr_reader :goal_width
    attr_reader :goal_height
    attr_reader :goal_depth
    attr_reader :goal_side_radius
    def initialize(json)
        @width = json["width"]
        @width = json["width"]
        @height = json["height"]
        @depth = json["depth"]
        @bottom_radius = json["bottom_radius"]
        @top_radius = json["top_radius"]
        @corner_radius = json["corner_radius"]
        @goal_top_radius = json["goal_top_radius"]
        @goal_width = json["goal_width"]
        @goal_height = json["goal_height"]
        @goal_depth = json["goal_depth"]
        @goal_side_radius = json["goal_side_radius"]
    end
end

File: packages/python3/model/arena.py
class Arena:
    def __init__(self, json):
        self.width = json.get("width")
        self.width = json.get("width")
        self.height = json.get("height")
        self.depth = json.get("depth")
        self.bottom_radius = json.get("bottom_radius")
        self.top_radius = json.get("top_radius")
        self.corner_radius = json.get("corner_radius")
        self.goal_top_radius = json.get("goal_top_radius")
        self.goal_width = json.get("goal_width")
        self.goal_height = json.get("goal_height")
        self.goal_depth = json.get("goal_depth")
        self.goal_side_radius = json.get("goal_side_radius")
